cli/pom.xml

```diff
@@ -90,6 +90,12 @@
       <artifactId>jansi</artifactId>
       <version>${jansi.version}</version>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.13.2</version>

```

Comment:
we already use JUnit 5 for our tests.
Why do you want to add JUnit 4 that is outdated?

Reply:
Import added this one automatically. I will change it to Junit5

Reply:
IntelliJ is smart but can also cause harm with too much magic ;)
Thanks for the update 👍